deploy.php

```diff
@@ -94,12 +94,13 @@
     run('{{bin/php}} {{release_path}}/bin/magento cache:enable');
 });
 
-desc('PHP Opcache flush');
+desc('PHP Opcache flush'); // JS This is for the Nexcess issue I think. Is this always necessary?
 task('php:opcache:flush', function() {
     $phpVersion = run('{{bin/php}} -r "echo phpversion();"');
     $cacheToolVersion = version_compare($phpVersion, '7.2', '>=') ? '' : '-3.2.1';
-
-    run('
+    
+    // JS Should we fork this to always have it availble in our repo at least?
```

Comment:
No and no ... mostly!

I would (being a bit paranoid) fork them into our own repo anyway just to have a copy in case they vanish off the internet but stilll reference and pull them from the actual repos to get the most up to date copies. 

Does that make sense?